packages/presets/src/ai/entries/format-bar/config.ts

```diff
@@ -240,7 +241,23 @@ const ReviewWIthAIGroup: AIItemGroupConfig = {
       name: 'Explain this image',
       icon: AIPenIcon,
       showWhen: imageBlockShowWhen,
-      handler: actionToHandler('explainImage'),
```

Comment:
I think explain image in doc already works.
The function was designed to work for multi-schema selections so that we can do an action with text + image combined. However this was changed to only include image if only a single image block is being selected